plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/impl/FeatureAttribute.java

```diff
@@ -220,66 +216,75 @@ public String getValue() {
 	 * @param Set configurable with a boolean value.
 	 */
 	public void setConfigurable(boolean configurable) {
-		this.configurable = configurable;
+		if (configurable) {
+			this.configurable = configurable;
+		}
```

Comment:
Why this if statement? This prevents the configurable element to be set to false!